test/integration/targets/win_module_utils/library/legacy_only_new_way_win_line_ending.ps1

```diff
@@ -1,6 +1,6 @@
-#!powershell
-
-#Requires -Module Ansible.ModuleUtils.Legacy
-
-Exit-Json @{ data="success" }
-
+#!powershell
```

Comment:
The line endings for this file was explicitly set to `\r\n` as part of the test, please keep it back to the old way.